src/PhpSpec/CodeAnalysis/TokenizedTypeHintRewriter.php

```diff
@@ -171,4 +171,38 @@ private function tokenHasType($token, $type)
     {
         return is_array($token) && $type == $token[0];
     }
+
+    /**
+     * Returns true if the function should be read with the rewriter.
+     *
+     * @param array $tokens
+     * @param int $index
+     *
+     * @return bool
+     */
+    private function functionShouldBeRead(array $tokens, $index)
+    {
+        return preg_match('/^(it|its)[^a-zA-Z]/', $this->getFunctionName($tokens, $index));
```

Comment:
Exactly here: https://github.com/phpspec/phpspec/blob/master/src/PhpSpec/Loader/ResourceLoader.php#L69

The "problem" is that just isolating this regex in a method might be too much.